app/api/route/issue_api.rb

```diff
@@ -53,11 +54,11 @@ def post_or_get_issue
           scope = scope.intersects(group.profile.location)
         end
         case params[:order]
-        when 'vote_count'
+        when :vote_count
           scope = scope.plusminus_tally
-        when 'created_at', 'start_at'
-          scope = scope.order(params[:order] => :desc)
-        when 'size'
+        when :created_at, :start_at, :id
+          scope = scope.order(params[:order] => params[:order_direction])
+        when :size
```

Comment:
see https://github.com/cyclestreets/cyclescape/commit/b28f1d2753fe0fc5ab2c8f4bbaa1ce74d26eacbf you can now call 
```ruby
scope.order_by_size(params[:order_direction])
```